src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop): Use abstract reference instead of specific one where possible: